SignalServiceKit/src/Messages/OWSMessageHandler.h

```diff
@@ -0,0 +1,24 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+extern const NSUInteger kIncomingMessageBatchSize;
+
+@class OWSSignalServiceProtosContent;
+@class OWSSignalServiceProtosDataMessage;
+@class OWSSignalServiceProtosEnvelope;
+
+NSString *envelopeAddress(OWSSignalServiceProtosEnvelope *envelope);
+
+@interface OWSMessageHandler : NSObject
```

Comment:
A common base class for the "message decrypter" and the "message manager".  Contains utility methods used for logging.